main.py

```diff
@@ -2,6 +2,36 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    genres_list_to_create = ["Western", "Action", "Dramma"]
+
+    for genre in genres_list_to_create:
+        Genre.objects.create(name=genre,)
+
+    actors_list_to_create = [("George", "Klooney"), ("Kianu", "Reaves"),
+                             ("Scarlett", "Keegan"), ("Will", "Smith"),
```

Comment:
split the actors by separate lines, each containing a single actor
this formatting makes it clearer and more readable, as each line represents an individual actor